### PR TITLE
fix(unlock-app) Rounding

### PR DESCRIPTION
**Pull request by julien51**
# Description

Fixing rounding UI issue.

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/helpers/BalanceProvider.js [hunk 1]
```diff
@@ -8,8 +8,9 @@ export const BalanceProvider = ({ amount, render }) => {
   if (typeof amount === 'undefined' || amount === null) {
     return render(' - ', ' - ') || null
   }
-  const currency = parseFloat(amount)
+  const currency = parseFloat(amount).toFixed(3).replace(/00$/, '')
   const ethWithPresentation = currency
+  console.log(typeof ethWithPresentation)
```
Comment (searchableguy) [inline unlock-app/src/components/helpers/BalanceProvider.js:13]: Remove the unnecessary log?

File: unlock-app/src/components/interface/verification/Key.tsx [hunk 2]
```diff
@@ -161,7 +161,10 @@ export const ValidKey = ({
 
   useEffect(() => {
     const onLoad = async () => {
-      if (!viewer) return
+      if (!viewer) {
+        setLoading(false)
+        return
+      }
```
Comment (searchableguy) [inline unlock-app/src/components/interface/verification/Key.tsx:167]: Didn't you fix this in another PR?

**Review by searchableguy — CHANGES_REQUESTED** [review]

File: unlock-app/src/components/interface/verification/Key.tsx (hunk 2 above)
Comment (julien51) [reply unlock-app/src/components/interface/verification/Key.tsx:167]: Yes, I need a rebase. sorry!

File: unlock-app/src/components/helpers/BalanceProvider.js (hunk 1 above)
Comment (julien51) [inline unlock-app/src/components/helpers/BalanceProvider.js:13]:
```suggestion
```

**Pull request merged**